python/grass/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+
+copy_python_files_in_subdir(bandref etc/python/grass)
+copy_python_files_in_subdir(exceptions etc/python/grass)
+copy_python_files_in_subdir(gunittest etc/python/grass)
+copy_python_files_in_subdir(imaging etc/python/grass)
+copy_python_files_in_subdir(pydispatch etc/python/grass)
+copy_python_files_in_subdir(script etc/python/grass)
+copy_python_files_in_subdir(temporal etc/python/grass)
+copy_python_files_in_subdir(pygrass etc/python/grass)
+copy_python_files_in_subdir(pygrass/messages etc/python/grass)
+copy_python_files_in_subdir(pygrass/modules etc/python/grass)
+copy_python_files_in_subdir(pygrass/raster etc/python/grass)
+copy_python_files_in_subdir(pygrass/vector etc/python/grass)
+copy_python_files_in_subdir(pygrass/gis etc/python/grass)
+copy_python_files_in_subdir(pygrass/shell etc/python/grass)
+copy_python_files_in_subdir(pygrass/tests etc/python/grass)
+copy_python_files_in_subdir(pygrass/rpc etc/python/grass)
+copy_python_files_in_subdir(pygrass/modules/interface etc/python/grass)
+copy_python_files_in_subdir(pygrass/modules/grid etc/python/grass)
+
+
+configure_file(script/setup.py
+  ${CMAKE_BINARY_DIR}/etc/python/grass/script/setup.py
+  COPYONLY)
+
+add_custom_target(LIB_PYTHON
+	COMMAND ${CMAKE_COMMAND} -E echo "#coding=utf-8" > ${GISBASE}/etc/python/__init__.py
+	COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py  ${GISBASE}/etc/python/grass/
+	COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/etc/python/grass/script/setup.py ${GISBASE}/etc/python/grass/script/
```

Comment:
The echo command adding an ut8 shebang breaks when running the build step. 
In the docs for add_custom_target, in the [section for option `VERBATIM`](https://cmake.org/cmake/help/latest/command/add_custom_target.html#:~:text=no%20build%20rules.-,VERBATIM,specific%20because%20there%20is%20no%20protection%20of%20tool%2Dspecific%20special%20characters.,-USES_TERMINAL), use of `VERBATIM` is recommended as it escapes properly for the build tool to see the argument inchanged as it is entered here. Without it, the CMake language processor eats one level of escapes, and all the commands to run are on one line, so the command to run is after `echo ` is a comment, breaking the rest of the line. 

Note:
_I added `--trace-source=/workspace/grass/python/grass/CMakeLists.txt` and `--trace-expand` to see what was being called_ 
```shell
gitpod /workspace/grass (cmake_build) $ cmake -v -S . -B build  --trace-source=/workspace/grass/cmake/modules/copy_python_files_in_subdir.cmake -Wno-dev --trace-source=/workspace/grass/python/grass/CMakeLists.txt --trace-source=/workspace/grass/gui/CMakeLists.txt
```
and saw the following at the end of the configure step (see the removed quotes on the echo command)
```shell
/workspace/grass/python/grass/CMakeLists.txt(22):  configure_file(script/setup.py ${CMAKE_BINARY_DIR}/etc/python/grass/script/setup.py COPYONLY )
/workspace/grass/python/grass/CMakeLists.txt(26):  add_custom_target(LIB_PYTHON COMMAND ${CMAKE_COMMAND} -E echo #coding=utf-8 > ${GISBASE}/etc/python/__init__.py COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py ${GISBASE}/etc/python/grass/ COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/etc/python/grass/script/setup.py ${GISBASE}/etc/python/grass/script/ DEPENDS python_bandref python_exceptions python_imaging python_pydispatch python_script python_temporal python_pygrass python_pygrass_messages python_pygrass_modules python_pygrass_modules python_pygrass_raster python_pygrass_vector python_pygrass_gis python_pygrass_shell python_pygrass_tests python_pygrass_rpc python_pygrass_modules_interface python_pygrass_modules_grid )
/workspace/grass/python/grass/CMakeLists.txt(40):  set_target_properties(LIB_PYTHON PROPERTIES FOLDER lib )
/workspace/grass/python/grass/CMakeLists.txt(42):  install(DIRECTORY ${GISBASE}/etc/python/grass DESTINATION etc/python/grass/ )
-- Configuring done
-- Generating done
-- Build files have been written to: /workspace/grass/build
gitpod /workspace/grass (cmake_build) $
``` 

So, I suggest adding the VERBATIM option, as in the docs:

> `VERBATIM`
> All arguments to the commands will be escaped properly for the build tool so that the invoked command receives each argument unchanged. Note that one level of escapes is still used by the CMake language processor before add_custom_target even sees the arguments. Use of `VERBATIM` is recommended as it enables correct behavior. When `VERBATIM` is not given the behavior is platform specific because there is no protection of tool-specific special characters.

```suggestion
	COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/etc/python/grass/script/setup.py ${GISBASE}/etc/python/grass/script/
	VERBATIM
```